File: OPENAPI-REST-API/swagger-client/dart/lib/model/server_row.dart
part of swagger.api;

class ServerRow {
  /* The id of the server. */
  String serverId = null;
/* The account lid of the server. */
  String accountLid = null;
/* The hostname of the server. */
  String serverHostname = null;
/* The status of the server. */
  String serverStatus = null;

  ServerRow();

  @override
  String toString() {
    return 'ServerRow[serverId=$serverId, accountLid=$accountLid, serverHostname=$serverHostname, serverStatus=$serverStatus, ]';
  }

  ServerRow.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    serverId = json['server_id'];
    accountLid = json['account_lid'];
    serverHostname = json['server_hostname'];
    serverStatus = json['server_status'];
  }

  Map<String, dynamic> toJson() {
    return {
      'server_id': serverId,
      'account_lid': accountLid,
      'server_hostname': serverHostname,
      'server_status': serverStatus
     };
  }

  static List<ServerRow> listFromJson(List<dynamic> json) {
    return json == null ? new List<ServerRow>() : json.map((value) => new ServerRow.fromJson(value)).toList();
  }

  static Map<String, ServerRow> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, ServerRow>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new ServerRow.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/ServerRow.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// A result row from the &#x60;Servers&#x60; &#x60;GET&#x60; request.
  /// </summary>
  [DataContract]
  public class ServerRow {
    /// <summary>
    /// The id of the server.
    /// </summary>
    /// <value>The id of the server.</value>
    [DataMember(Name="server_id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "server_id")]
    public string ServerId { get; set; }

    /// <summary>
    /// The account lid of the server.
    /// </summary>
    /// <value>The account lid of the server.</value>
    [DataMember(Name="account_lid", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "account_lid")]
    public string AccountLid { get; set; }

    /// <summary>
    /// The hostname of the server.
    /// </summary>
    /// <value>The hostname of the server.</value>
    [DataMember(Name="server_hostname", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "server_hostname")]
    public string ServerHostname { get; set; }

    /// <summary>
    /// The status of the server.
    /// </summary>
    /// <value>The status of the server.</value>
    [DataMember(Name="server_status", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "server_status")]
    public string ServerStatus { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ServerRow {\n");
      sb.Append("  ServerId: ").Append(ServerId).Append("\n");
      sb.Append("  AccountLid: ").Append(AccountLid).Append("\n");
      sb.Append("  ServerHostname: ").Append(ServerHostname).Append("\n");
      sb.Append("  ServerStatus: ").Append(ServerStatus).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
